koku/api/report/aws/aws_query_handler.py

```diff
@@ -50,12 +51,27 @@ def __init__(self, query_parameters, url_data,
             tenant    (String): the tenant to use to access CUR data
             kwargs    (Dict): A dictionary for internal query alteration based on path
         """
-        if not kwargs.get('provider'):
-            kwargs['provider'] = 'AWS'
+        provider = 'AWS'
```

Comment:
I've simplified this to being a simple variable because it's mainly used as part of the logic for selecting the ProviderMap. So, it's a bit extraneous, but I left it mostly alone because I didn't want to include a refactor of the map-selection logic in the ProviderMap.